fighters/mario/src/opff.rs

```diff
@@ -274,6 +274,12 @@ unsafe fn fastfall_specials(fighter: &mut L2CFighterCommon) {
     }
 }
 
+pub unsafe fn hhhhhhh(fighter: &mut L2CFighterCommon) {
+    if fighter.is_button_on(Buttons::Parry) {
+        println!("the parry");
```

Comment:
Lingering printlns be like

Reply:
Quick on the draw, I was just about to remove that